react/queries/orderForm.graphql

```diff
@@ -1,7 +1,8 @@
 # import '../fragments/orderForm.graphql'
 
-query orderForm {
-  orderForm @context(scope: "private") {
+query orderForm($refreshOutdatedData: Boolean) {
+  orderForm(refreshOutdatedData: $refreshOutdatedData)
```

Comment:
Just to make sure: omitting the `orderFormId` here won't be a problem in the future, right?

Reply:
it won't, that's only for stores that don't use the checkout cookie